#KEY#DATE# 2025-03-11
#KEY#MAX_DIFF_LINES# 100
#KEY#EXCLUDES# pnpm.lock, TypeScript build files, Rust build files

#KEY#START_COMMIT#
=================================================================
#KEY#COMMIT_HASH# 5f1a5611cfaff3aeb4b264d3adc14e58d02413b8
#KEY#COMMIT_DATE# 2025-03-11 05:32:18 -0700
#KEY#COMMIT_AUTHOR# Malte Ubl
#KEY#COMMIT_MESSAGE# Remove routing to / (#1)
#KEY#RELATED_PR_MR# None found

#KEY#CHANGED_FILES#
- README.md

#KEY#DIFF_RANGE# FROM: 51e16c30846fdd81236219afc94971006a027dd7 TO: 5f1a5611cfaff3aeb4b264d3adc14e58d02413b8
=================================================================

#KEY#GIT_DIFF#

diff --git a/README.md b/README.md
index 0105894..9ddc123 100644
--- a/README.md
+++ b/README.md
@@ -1,22 +1,20 @@
 # Run an MCP Server on Vercel
 
 ## Usage
 
 Update `api/server.ts` with your tools, prompts, and resources following the [MCP TypeScript SDK documentation](https://github.com/modelcontextprotocol/typescript-sdk/tree/main?tab=readme-ov-file#server).
 
-[There is also a Next.js version of this template](https://vercel.com/templates/next.js/model-context-protocol-mcp-with-next-js)
-
 ## Notes for running on Vercel
 
 - Requires a Redis attached to the project under `process.env.REDIS_URL`
 - Make sure you have [Fluid compute](https://vercel.com/docs/functions/fluid-compute) enabled for efficient execution
 - After enabling Fluid compute, open `vercel.json` and adjust max duration to 800 if you using a Vercel Pro or Enterprise account
 - [Deploy the MCP template](https://vercel.com/templates/other/model-context-protocol-mcp-with-vercel-functions)
 
 ## Sample Client
 
 `script/test-client.mjs` contains a sample client to try invocations.
 
 ```sh
 node scripts/test-client.mjs https://mcp-on-vercel.vercel.app
 ```
#KEY#END_COMMIT#
